tests/circuit_graph/test_qasm.py

```diff
@@ -98,6 +98,38 @@ def test_unused_wires(self):
 
         assert res == expected
 
+    def test_to_ApproxTimeEvolution(self):
+        """Test for case that requires a decomposition depth of 3 to successfully convert to a MutliRZ"""
+        H = qml.Hamiltonian(
+            [
+                1,
+            ],
+            [
+                qml.PauliZ(0) @ qml.PauliZ(1),
+            ],
+        )
+        with qml.queuing.AnnotatedQueue() as q_circuit:
+            qml.ApproxTimeEvolution(H, 1, n=1)
+
+        circuit = qml.tape.QuantumScript.from_queue(q_circuit)
```

Comment:
```suggestion
        circuit = qml.tape.QuantumScript([qml.ApproxTimeEvolution(H, 1, n=1)])
```

Non-blocking change, but it is usually tidier to avoid queuing if possible. 

The other tests here are from before we separated queuing from representation, so they just have the minimal changes required to port them over, instead of just using the now recommended syntax.